bin/release.sh

```diff
@@ -3,22 +3,34 @@
 
 set -e
 
-if [[ $1 == "" ]]; then
-    echo "No release version set"
-    exit 1
-fi
-
 RELEASE=$1
 
-if ! [[ $RELEASE =~ ^([0-9.]+)$ ]]; then
-    echo "Release tag does not match expected pattern: MAJOR.MINOR.PATCH"
-    exit 1
-fi
-
-if ! [[ -z $(git status -s) ]]; then
-    echo "You have uncommited or staged changes on git, please commit them or stash them"
-    exit 1
-fi
+function isGitClean {
+    if ! [[ -z $(git status -s) ]]; then
+        echo "You have uncommited or staged changes on git, please commit them or stash them"
+        return 1
+    fi
+    return 0
+}
+
+function displayVersioningHint {
+    echo "Versions altered, you may fix this by running the following:"
+    echo ""
+    echo "  git add -u && git commit --amend --no-edit -a && bin/release.sh $RELEASE "
+    echo ""
+    echo "When preparing a new release, you can adit all package.json files at once via"
```

Comment:
Just a minor typo on "...*edit* all package.json..." :)

Reply:
didn't you know, _adit_ is slang for _audit_ and _edit_ 😄 
jk -fixed it